README.md

```diff
@@ -1 +1,151 @@
 # ampersand-multifield-view
+
+## purpose
+A view module for intelligently grouping multiple field views. Works well with [ampersand-form-view][ampersand-form-view].
```

Comment:
How about:

A view module for intelligently applying a hierarchy to form data with the possibility of its own template and logic. It reports its fields to its parent in the form of an object where the object's keys and values are the fields' `name`s and `value`s. Works with with ampersand-form-view.